gh2/csv.py

```diff
@@ -56,8 +56,13 @@ def make_parser():
         help='Only include issues with this label in the output. Multiple '
              'invocations of this flag will require the issue to have all '
              ' labels.',
-        action='append', dest='filter_labels',
-        default=[],
+        action='append', dest='filter_labels', default=[],
+    )
+    args.add_argument(
```

Comment:
well, it's not common at the moment as you can't yet do it :) But you make a good point. I was following the pattern of 'additional labels' which are optional and off by default, but I really can't see the harm in including the milestone in the output for all runs and just removing the option completely.